api/src/routers/verification/index.ts

```diff
@@ -274,6 +316,8 @@ verificationRouter.post('/check-credential', apiKeyMiddleware, validate({ body:
  *         application/json:
  *           schema: 
  *             $ref: "#/components/schemas/RevokeVerificationBodySchema"
+ *           example: 
+ *             signatureValue: example-key
```

Comment:
please use this as example:
`"signatureValue": "8VZJmBRcjjsmzoLGcdap5MWBqgKVf27Gbvubatk2ibxU.1112Pe57T4J3dzcCs5V2X2ffM8Xex521NkscDDoa8cshh7sSM.596PBh54fsBnHj9P5Fgw3mSJrChFNHDifb2mf4oK6kMBL8PU52c5f5oMHCrV3CudZ9PoexjRfrjL6TjAM22vkyYv"`